spec/bump/commands/deploy_spec.rb

```diff
@@ -14,7 +14,7 @@
         specification: 'openapi/v2/json',
         validation: 'strict',
         'auto-create': true)
-    end.to output(/New version has been successfully deployed/).to_stdout
+    end.to output(/New version is being processed/).to_stdout
```

Comment:
What do you think about "The new version is currently being processed"?

Reply:
:+1: